build.cake

```diff
@@ -297,6 +294,7 @@ Task("DotnetPack")
 ///  currently.
 /// </summary>
 Task("DotnetPackNuspec")
```

Comment:
Suggest updating task name to `NugetPackNuspec` as well if you change the name of the called method too